test/utils/PathUtils.test.ts

```diff
@@ -1,3 +1,4 @@
+import '../testhelpers/loggerSpy'; // side-effect: silences logs
```

Comment:
It looks like git got confused when merging these changes. I don't think we need this line since it's also included at the bottom of the import list. I'll let you decide which place you like it better though!

Reply:
Hmmm... I kind of like it on line 1.  Plus that gives me more git cred.  But at the same time, @ngfreiter got his in first, so it would be kind of silly to have this PR do nothing in that file except move the import line so... I guess I'll let Nick's stay. :-(